### PR TITLE
WIP add clickable pathways that lead to filtered schedule view

### PR DESCRIPTION
**Pull request by ryanpitts**

### CODE REVIEW

**Comment by ryanpitts** [comment]
So far, this takes the pathway list for a session, breaks it into an array and renders links for each one.

screen shot 2015-10-29 at 3 00 17 pm

The links target a location that starts with `#_pathway` and then adds a slugified version of the pathway name that we can match against. E.g: #_pathway-scaling-participatory-learning-experiences

File: js/schedule.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ function Schedule(options) {
         // currently this app supports two types of detail pages:
         // 1) _session (which gets a detail page for a given session)
         // 2) _show (which gets a session list for a specific tab)
-console.log(pageType, pageID);
+        //console.log(pageType, pageID);
```
Comment (mmmavis) [inline js/schedule.js]: heh, i'll have those temp `console.log` removed in my PR :wink:

**Comment by mmmavis** [comment]
Wow! Work like a charm! r+

(I have to admit I didn't check "the underscore.string formatters" thoroughly / line-by-line :grin: )

**Comment by ryanpitts** [comment]
OK, @mmmavis has reviewed, going to merge this one.

**Pull request merged**